qiskit/providers/ibmq/api/clients/websocket.py

```diff
@@ -82,13 +84,14 @@ class WebsocketClient(BaseClient):
         websocket_url (str): URL for websocket communication with IBM Q.
         access_token (str): access token for IBM Q.
     """
+    BACKOFF_MAX = 8  # Maximum time to wait between retries.
 
     def __init__(self, websocket_url: str, access_token: str) -> None:
         self.websocket_url = websocket_url.rstrip('/')
         self.access_token = access_token
 
     @asyncio.coroutine
-    def _connect(self, url: str) -> Generator[Any, None, Connect]:
+    def _connect(self, url: str) -> Generator[Any, None, WebSocketClientProtocol]:
```

Comment:
Can you also update the docstring for the return type (from `Connect` to `WebSocketClientProtocol`)?